### ISSUE TITLE
Bug: Duplicated cache key on assets loading

### ISSUE THREAD

**Issue by danielbarion** [pixijs/pixijs#11055]
### Current Behavior

Today the assets loader creates a few "duplicated" keys for each loaded asset/bundle:

![Image](https://github.com/user-attachments/assets/bbf94699-ed00-493f-9f80-d89965d779c9)

![Image](https://github.com/user-attachments/assets/5779138e-ef7e-4db7-8c26-f343b0746d0b)

The issue is that even if my assets are organized if the file has the same name inside different spritesheets, the Assets loader gives the warning.

![Image](https://github.com/user-attachments/assets/0ca913ed-38ff-42a7-b97a-aa0b06e2012d)

![Image](https://github.com/user-attachments/assets/a574bf4e-b406-40db-b6c6-7038e171f121)
![Image](https://github.com/user-attachments/assets/652b6416-dc77-4b09-85ba-0e6a03897793)

Files:

https://gist.githubusercontent.com/danielbarion/e5b32f93a766643cd7d2035d118d3bf7/raw/6503338e2862c38d08e08541b082d8cfd7789f76/priest.json

https://gist.githubusercontent.com/danielbarion/e5b32f93a766643cd7d2035d118d3bf7/raw/6503338e2862c38d08e08541b082d8cfd7789f76/pyromancer.json

### Expected Behavior

.

### Steps to Reproduce

.

### Environment

- **`pixi.js` version**: *8.5.2* - happens since v7 (I already talked about it in the past on Discord)
- **Browser & Version**: *Version 130.0.6723.92 (Official Build) (arm64)*

**Comment by danielbarion** [comment]
@lunarraid not sure if you remember we talked about it in the past, but I finally created the bug ticket 🙏🏻

**Comment by lunarraid** [comment]
@danielbarion This was already done for v7 but never ported to v8. I assume a similar fix can be done here.

https://github.com/pixijs/pixijs/pull/10125

**Comment by bigtimebuddy** [comment]
Yeah, I was gonna say, I remember solving this already! We should add cachePrefix to v8.

**Comment by danielbarion** [comment]
That sounds great to me, thanks guys! I didn't know it was solved on V7, I'm using V8 since the V8 release, thanks!

**Comment by lunarraid** [comment]
@danielbarion This change would need to specifically remove the second key (index 1) of necessity, so nope.

**Comment by marcoshier** [comment]
Has anybody found a workaround for this? It is horrible to have hundreds of Warning messages in console and absolutely no way to disable them.

**Comment by fennean1** [comment]
Hey folks: 

Any updates on this for version 8?


From what I can tell, it's just creating duplicate keys within the textures but the graphics panel only shows a single texture. I'm hoping this means that I'm not taking up extra memory with duplicate keys but it would be really great to have a fix for this. 

Are there any updates? Claude mentioned something about setting bundlePrefix = false but that didn't work for me.

**Comment by Zyie** [comment]
This should now be fixed in 8.10.0

**Comment by danielbarion** [comment]
@Zyie just a heads-up, I only had time to upgrade and check it now (I'm sorry for the delay),

I'm still receiving the warnings:

<img width="993" height="880" alt="Image" src="https://github.com/user-attachments/assets/4d781ddd-a6a2-44df-9120-a21308b37573" />
